### DISCUSSION TITLE
How can i make a Sprite from a SpriteBundle sized in World Units?

### DISCUSSION THREAD

**Discussion by DasLixou** [bevyengine/bevy#6512] [Q&A] [answered]
Is there a way to make a sprite in size of world units? like 1 or 5 world units. the same that are used in ScalingMode::Auto?

Thanks :D

**Comment by tim-blackbird** [comment, marked as answer]
The `Sprite` component has a `custom_size` field.
https://docs.rs/bevy/latest/bevy/prelude/struct.Sprite.html

**Reply by DasLixou** [reply]
So when i use Some(Vec2::splat(1.0)) there it will be 1 world unit?

**Reply by tim-blackbird** [reply]
Yep
